specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json

```diff
@@ -286,6 +286,14 @@
           "description": "Preserve Rules."
         }
       },
+      "preserve": {
```

Comment:
Doesn't look like the tools pick it up, but in VS Code this gets flagged as "Property preserve is not allowed.", but I didn't see anything in the JSON schema for that

Reply:
Ok, let me check again